include/ccmath/math/nearest/ceil.hpp

```diff
@@ -8,7 +8,62 @@
 
 #pragma once
 
+#include "ccmath/math/nearest/floor.hpp"
+
+#include <type_traits>
+
 namespace ccm
 {
+	namespace internal::impl
+	{
+		template <typename T, std::enable_if_t<std::floating_point_v<T>, bool> = true>
```

Comment:
Forgot the `is` in `is_floating_point_v`

```suggestion
		template <typename T, std::enable_if_t<std::is_floating_point_v<T>, bool> = true>
```